index-advisor/getting-started.mdx

```diff
@@ -113,8 +113,8 @@ sometimes indexes are created but never used, or cease to be used. This can
 happen either because a better index is available, the current index cannot be
 used even though it looked applicable, or the query workload changed.
 
-The unused index check detects indexes that haven't been used in the last 2
-weeks, as determined by Postgres statistics, and notes these indexes as
+The unused index check detects indexes that haven't been used in the last 35
+days, as determined by Postgres statistics, and notes these indexes as
```

Comment:
Should we note that this is the default? i.e. like

```
The unused index check detects indexes that haven't been used recently (last 35 days by default), as determined by Postgres statistics,
```

We could also link "unused index check" to the check page at https://pganalyze.com/docs/checks/schema/index_unused